tests/ops/integration_tests/saas/test_braze_task.py

```diff
@@ -166,7 +171,7 @@ async def test_saas_erasure_task(
         f"{dataset_name}:subscription_groups_email": 0,
     }
 
-    time.sleep(10)
+    sleep(30)
```

Comment:
as you mentioned to me offline, introducing a new polling function, or enhancing `poll_for_existence`, to support verifying masked fields, would be nice here.

but i get how that's out of scope of the PR here. so maybe we make an issue to follow up on this?